java/enum.go

```diff
@@ -2,31 +2,46 @@ package java
 
 import (
 	"fmt"
+	"github.com/zocdoc/gopoetry/util"
 	"strings"
 )
 
 // ClassDeclaration declares a class
 type EnumDeclaration struct {
 	name        string
 	enumMembers map[string]string
-	members     []Writable
+	members     []util.Writable
+	modifiers   []string
+}
+
+func (cls *EnumDeclaration) addModifier(modifier string) *EnumDeclaration {
```

Comment:
This change snuck in. Sorry. Allows java methods to have a modifier like `private` or `public`